user/homework.go

```diff
@@ -32,6 +33,7 @@ const (
 type Homework struct {
 	Items    []*Item
 	template string
+	width    int
```

Comment:
Also, I don't want to be printing output anywhere except in the cmd package. This fmt.Print stuff is so dirty here.